docs/infrastructure/metrics-explorer.asciidoc

```diff
@@ -0,0 +1,59 @@
+[role="xpack"]
+[[metrics-explorer]]
+== Using the Metrics explorer
+
+The metrics explorer allows you to create system load charts for each host you're monitoring with Metricbeat.
```

Comment:
Is there anything else we can add here that explains "why" someone would use this page?

Reply:
I change this text to be a little more generic since the metric explorer can do more then create charts for "system load". I'm open to more suggestions, at the moment I'm having a hard time articulating the "why".